client/src/components/Profile/Profile.js

```diff
@@ -1,17 +1,64 @@
+import { useEffect, useState } from 'react'
+import axios from 'axios'
 import styled from 'styled-components/macro'
 import LinkButtonBlue from '../Buttons/LinkButtonBlue/LinkButtonBlue'
-import LinkButtonGreen from '../Buttons/LinkButtonGreen/LinkButtonGreen'
+import ButtonGreen from '../Buttons/ButtonGreen/ButtonGreen'
```

Comment:
I like that you renamed your component. Maybe for consistency it would be better to rename `LinkButtonBlue` & `LinkButtonRed` too. :-)